proxy/ControlBase.cc

```diff
@@ -423,12 +422,12 @@ TextMod::print(FILE *f) const
 void
 TextMod::set(const char *value)
 {
-  free(this->text.data());
-  this->text.set(ats_strdup(value), strlen(value));
```

Comment:
With regard to the previous comment, if the string is being duplicated already, why not `std::string`?